app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -97,7 +100,15 @@ private fun HomeScreenContent(
 
                 item {
                     PortfolioCard(
-                        modifier = Modifier.padding(start = Dp16, top = Dp40, end = Dp16)
+                        modifier = Modifier
+                            .padding(start = Dp16, top = Dp40, end = Dp16)
+                            .boxShadow(
+                                borderRadius = 3.dp,
+                                color= Color.TiffanyBlue,
+                                blurRadius = 19.dp,
+                                offsetY = 212.dp,
+                                spread = -2.1f
```

Comment:
It would help much if you could share the details about the values in here, how could we have that values? 😄

Reply:
Yeah sure ,

 

**borderRadius :** Can controll border radius of shadow
 **color :** Color of shadow of Portfilo Card 
**blurRadius :** help to change blur of Shadow-Color
**offsetY :** we change the position of the shadow according to Y axis
**offsetX :** we change the position of the shadow according to X axis
**spead :** we can squeeze or release the shadow of Portfilo Card

Reply:
@wasim15185 Thank you for your clarifying 👍